src/Tests/Microsoft.Diagnostics.Monitoring.OpenApiGen/Program.cs

```diff
@@ -50,6 +50,7 @@ public static void Main(string[] args)
                     diagnosticPort: null,
                     noAuth: false,
```

Comment:
I wonder at what point we make this some kind of "bag of flags" to avoid having a method with 100 parameters. To be clear, I don't think this should be done now, but if we keep adding more of these we may want to find a better way.

Reply:
The System.CommandLine library mostly forces this pattern, but at lower levels (which I think CreateHostBuilder method can be considered at a lower level) we could move to using a settings object (a configuration object for setting up the configuration, if you will :smile:).